packages/core/tests/integration/components/field-renderer-test.js

```diff
@@ -147,17 +150,76 @@ module('Integration | Component | field-renderer', function(hooks) {
       @setNeededWhenEmbedded={{action noop}}
       @setPosition={{action noop}}
       @removeField={{action noop}}
+      @isRightEdgeField={{true}} {{!-- Burcu please delete this line when you merge your stuff --}}
+      @schemaAttrs={{array "title" "type" "label" "name" "instructions" "is-meta" "embedded" "required"}}
     />
     `);
 
-    assert.dom('[data-test-field-renderer-type]').hasText('@cardstack/core-types::string');
-    assert.dom('[data-test-field-renderer-label]').hasText('title');
-    assert.dom('[data-test-field-renderer-value]').hasText('test title');
+    assert.dom('.schema-field-renderer--header--detail').doesNotExist();
+    assert.dom('[data-test-field-renderer-field-type]').hasText('@cardstack/core-types::string');
     assert.dom('[data-test-field-renderer-move-btn]').exists();
     assert.dom('[data-test-drop-zone="1"]').exists();
 
     assert.dom('.edit-title-field-value').doesNotExist();
     assert.dom('[data-test-string-field-viewer-value]').doesNotExist();
+
+    assert.dom('[data-test-schema-attr="name"] input').hasValue('title');
+    assert.dom('[data-test-schema-attr="name"] input').isNotDisabled();
+    assert.dom('[data-test-schema-attr="label"] input').hasValue('title');
+    assert.dom('[data-test-schema-attr="label"] input').isNotDisabled();
+    assert.dom('[data-test-schema-attr="instructions"] textarea').isNotDisabled();
+    assert.dom('[data-test-schema-attr="embedded"] input').isChecked();
+    assert.dom('[data-test-schema-attr="embedded"] input').isNotDisabled();
+  });
+
+  test("it can render an adopted field in schema mode", async function (assert) {
+    // name, label, and needed-when-embedded should be disabled
+    // instructions and required should be enabled
+
+    let service = this.owner.lookup('service:data');
+    let parent = service.createCard(qualifiedCard1Id);
+    parent.addField({ name: 'body', type: '@cardstack/core-types::string', neededWhenEmbedded: true });
+    parent.addField({ name: 'title', type: '@cardstack/core-types::string', neededWhenEmbedded: true });
+    parent.addField({ name: 'author', type: '@cardstack/core-types::string', neededWhenEmbedded: true });
+    await parent.save();
+
+    let card = service.createCard(qualifiedCard2Id, parent);
+    let field = card.getField('title');
+    field.setValue('test title')
+    this.set('field', field);
+    this.set('noop', () => {});
+
+    await render(hbs`
+    <FieldRenderer
+      @field={{field}}
+      @mode="schema"
+      @dropField={{action noop}}
+      @setFieldName={{action noop}}
+      @setFieldLabel={{action noop}}
+      @setFieldInstructions={{action noop}}
+      @setNeededWhenEmbedded={{action noop}}
+      @setPosition={{action noop}}
+      @removeField={{action noop}}
+      @isRightEdgeField={{true}} {{!-- Burcu please delete this line when you merge your stuff --}}
```

Comment:
@burieberry Please make sure to delete this line from the tests after you rebase your work to remove the `@isRightEdgeField`